modules/swagger-codegen/src/main/resources/Java/StringUtil.mustache

```diff
@@ -48,4 +48,12 @@ public class StringUtil {
     if (o == null) return "null";
     return o.toString().replace("\n", "\n    ");
   }
+
+  public static boolean isNotBlank(String s){
```

Comment:
Yea my implementation isn't comprehensive .... I looked at this implementation but was trying to eliminate further class dependencies i.e. Character, CharacterData, etc.